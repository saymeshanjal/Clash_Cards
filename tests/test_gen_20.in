90
6 294 1
4 143 5
2 330 5
12 240 3
10 173 2
12 333 2
16 235 3
6 164 4
14 115 2
14 217 4
8 211 1
2 101 4
16 197 1
10 338 2
12 343 2
10 92 3
10 355 3
12 189 2
8 387 4
16 50 2
2 77 5
2 397 1
2 109 1
4 321 1
8 278 1
8 226 3
2 191 5
2 60 4
10 367 4
10 238 4
12 143 1
4 279 1
8 39 2
6 57 5
8 172 4
10 111 2
2 207 3
2 295 3
10 140 1
10 160 4
14 221 4
14 38 4
8 113 5
12 243 4
2 271 4
6 139 5
16 291 3
6 283 5
6 256 5
4 196 2
14 155 3
8 161 4
16 164 2
6 273 3
16 313 4
2 212 5
2 147 3
2 261 2
2 263 3
8 24 4
16 279 1
16 274 4
14 289 3
16 363 3
8 106 2
14 394 1
10 139 3
12 347 1
6 115 5
16 383 1
4 14 2
12 77 1
8 116 2
10 243 1
16 350 4
8 164 1
8 272 2
14 69 4
14 288 5
14 335 5
16 56 4
14 310 3
2 57 1
6 213 4
2 204 1
4 343 5
10 42 4
6 200 1
8 327 4
2 141 1
