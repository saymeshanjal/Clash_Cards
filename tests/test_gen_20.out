10 10 0
0 6 0
0 2 0
430 494 0
76 77 99
286 287 351
6307 6563 0
1 14 5
1078 1079 1275
910 1520 1002
35 35 0
0 2 0
6435 6435 0
76 77 99
286 287 351
110 142 0
110 142 0
286 287 351
8 42 20
951 1479 10440
0 2 0
1 1 0
1 1 0
3 3 0
35 35 0
27 43 0
0 2 0
0 2 0
44 133 75
44 133 75
462 462 0
3 3 0
20 21 29
1 19 0
8 42 20
76 77 99
0 2 0
0 2 0
126 126 0
44 133 75
910 1520 1002
89 2443 900
8 62 0
208 441 275
0 2 0
1 19 0
6307 6563 0
1 19 0
1 19 0
1 2 3
1652 1780 0
8 42 20
4081 4082 4707
6 14 0
3809 5406 3655
0 2 0
0 2 0
0 1 1
0 2 0
7 42 21
6435 6435 0
3809 5406 3655
1652 1780 0
6307 6563 0
20 21 29
1716 1716 0
110 142 0
462 462 0
1 19 0
6435 6435 0
1 2 3
462 462 0
20 21 29
126 126 0
3809 5406 3655
35 35 0
20 21 29
910 1520 1002
987 2445 0
987 2445 0
495 7335 5040
1652 1780 0
1 1 0
1 14 5
1 1 0
0 6 0
44 133 75
10 10 0
8 42 20
1 1 0
